src/modules/media/services/media.service.ts

```diff
@@ -377,6 +377,9 @@ export class MediaService {
 		delete copy.meemooMediaObjectId;
 		delete copy.dateCreatedLowerBound;
 
+		// https://meemoo.atlassian.net/browse/ARC-1109?focusedCommentId=34838
+		delete copy.premisIsPartOf;
```

Comment:
seems cleaner to do both at the same position, but it's Personal Preference
```suggestion
		copy.PID = mediaObject.premisIsPartOf;
		delete copy.premisIsPartOf;
```